src/VirtoCommerce.SitemapsModule.Web/Controllers/Api/SitemapsModuleApiController.cs

```diff
@@ -412,11 +477,20 @@ void SendNotificationWithProgressInfo(ExportImportProgressInfo c)
         private async Task CreateSitemapPartAsync(ZipArchive zipArchive, string storeId, string baseUrl, string sitemapUrl, Action<ExportImportProgressInfo> progressCallback)
         {
             var sitemapPart = zipArchive.CreateEntry(sitemapUrl, CompressionLevel.Optimal);
-            using (var sitemapPartStream = sitemapPart.Open())
-            {
-                var stream = await _sitemapXmlGenerator.GenerateSitemapXmlAsync(storeId, baseUrl, sitemapUrl, progressCallback);
-                await stream.CopyToAsync(sitemapPartStream);
-            }
+            using var sitemapPartStream = sitemapPart.Open();
+            using var stream = await _sitemapXmlGenerator.GenerateSitemapXmlAsync(storeId, baseUrl, sitemapUrl, progressCallback);
+            await stream.CopyToAsync(sitemapPartStream);
+        }
+
+        private async Task<BlobInfo> ExportSitemapPartAsync(string storeId, string storeUrl, string outputAssetFolder, string sitemapUrl, Action<ExportImportProgressInfo> progressCallback)
+        {
+            var relativeUrl = $"{outputAssetFolder.Trim('/')}/{sitemapUrl.Trim('/')}";
```

Comment:
The same as in SitemapProcessJob.cs - if we work with real url link we may use Uri class for better experience and management.
Secondly, we basically have the same code for this function in two places (here and in SitemapProcessJob)